src/frontend/src/pages/ChangeRequestDetailPage/OtherChangeRequestsPopupTabs.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import React, { useState } from 'react';
+import { Box, useTheme, Collapse, Tabs, Tab, Typography } from '@mui/material';
+import { ChangeRequest } from 'shared';
+import ChangeRequestDetailCard from '../../components/ChangeRequestDetailCard';
+import { useAllChangeRequests } from '../../hooks/change-requests.hooks';
+import { ExpandLess, ExpandMore } from '@mui/icons-material';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+
+interface OtherChangeRequestsPopupTabsProps {
+  changeRequest: ChangeRequest;
+}
+
+const OtherChangeRequestsPopupTabs: React.FC<OtherChangeRequestsPopupTabsProps> = ({
+  changeRequest
+}: OtherChangeRequestsPopupTabsProps) => {
+  const theme = useTheme();
+  const [tab, setTab] = useState(0);
+  const { data: changeRequests, isError, isLoading, error } = useAllChangeRequests();
+
+  if (isLoading) return <LoadingIndicator />;
+  if (isError) return <ErrorPage error={error} message={error.message} />;
+
+  // the CRs submitted or reviewed by the submitter of this CR
+  const crFromSameUser = changeRequests?.filter(
```

Comment:
Can we change this variable name to `crsFromSubmitter` to be more clear which CRs are these from?